custom_components/multiscrape/util.py

```diff
@@ -5,23 +5,22 @@
 from homeassistant.exceptions import TemplateError
 from homeassistant.helpers.template import Template
 
-
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
 
 def create_renderer(hass, value_template):
     """Create a template renderer based on value_template."""
     if value_template is None:
-        return lambda value=None, parse_result=None: value
+        return lambda value=None, parse_result=None, variables={}: value
 
     if not isinstance(value_template, Template):
         value_template = Template(value_template, hass)
     else:
         value_template.hass = hass
 
-    def _render(value: Any = None, parse_result=False):
+    def _render(value: Any = None, parse_result=False, variables: dict = {}):
```

Comment:
Would it be cleaner to rename the `value` parameter to `values` or `variables` and check if it is a dictionary? If not, keep old behavior, other just provide variables and assume the dictionary has a value.
Your solution is more explicit but value and variables are actually both variables. What do you think?

Reply:
I decided to go with the option to rename value to variables, as I couldn't find usages of `value`. I guess it used before and usage stopped after a while.